CODEOWNERS

```diff
@@ -43,25 +43,20 @@
 /cpu/avr8_common/                           @kYc0o @maribu @nandojve
 /cpu/atmega*/                               @kYc0o @maribu
 /cpu/atxmega/                               @nandojve
-/cpu/cc2538/                                @hexluthor @smlng @fjmolinas
-/cpu/cc26x0/                                @hexluthor @smlng
-/cpu/cc26x2_cc13x2/                         @hexluthor @smlng
-/cpu/cc26xx_cc13xx/                         @hexluthor @smlng
 /cpu/cortexm_common/                        @kaspar030 @bergzand
 /cpu/efm32/                                 @basilfx
 /cpu/esp*/                                  @gschorcht
 /cpu/fe310/                                 @aabadie @kaspar030 @bergzand
-/cpu/kinetis/                               @fjmolinas
 /cpu/lpc2387/                               @benpicco @maribu
 /cpu/msp430*/                               @kaspar030 @gschorcht
-/cpu/nrf*/                                  @aabadie @haukepetersen
+/cpu/nrf*/                                  @aabadie
 /cpu/nrf52/radio/nrf802154/                 @bergzand @SemjonKerner @jia200x
```

Comment:
@SemjonKerner can also be removed

Reply:
He/she seems to even have removed his/her github account: https://github.com/SemjonKerner